ting_marc.module

```diff
@@ -346,3 +354,31 @@ function ting_marc_field_js() {
 
   drupal_json_output($result);
 }
+
+/**
+ * AJAX handler for author field load.
+ *
+ * @return array
+ *   Set of result authors by entity id.
+ */
+function ting_marc_authors_js() {
+  $result = array();
+
+  if (!empty($_POST['marc_authors'])) {
+    $ids = $_POST['marc_authors'];
+
+    foreach ($ids as $id) {
+      $object = ting_marc_load_object($id);
```

Comment:
Is there a need to request this function for each ID? Wouldn't it make sense to create a function that can handle this with a bulk of IDs using `search` instead of `getObject`?